readthedocs/builds/utils.py

```diff
@@ -16,6 +18,14 @@
     re.compile('bitbucket.org:(.+)/(.+)\.git'),
 ]
 
+# TODO: I think this can be different than `gitlab.com`
+# self.adapter.provider_base_url
+GL_REGEXS = [
+    re.compile('gitlab.com/(.+)/(.+)(?:\.git){1}'),
```

Comment:
Are we going to support private gitlab instalation? How this should looks like?

Reply:
We will, but this integration will be on the .com, we won't put that code here yet.

Reply:
OK. So, I think the TODO note is good for now. I will leave it there